workers/unity/Assets/Playground/Scripts/Player/PlayerCommandsSystem.cs

```diff
@@ -39,6 +41,20 @@ protected override void OnCreate()
 
         protected override void OnUpdate()
         {
+            if (lastId.HasValue)
```

Comment:
Do we want to add this to playground?
This might be rather spammy in a cloud build

Reply:
It only logs when you left/right-click, but if its still too spammy I can remove it. @jamiebrynes7 @paulbalaji what do you think?

Reply:
It might be useful locally, but don't think it's worth keeping for cloud builds